Tapestry-Inc/design-tokens
# Update: TORO-61173 Star Icon 1px Difference

Ticket: https://jira.tapestry.support/browse/TORO-61173 <br>

## Changes

src/global/asset/icon/review/star.svg
> - Replaced old icon with new icon recently exported from Figma.  This new icon has the same path values as the other star icons.
---

## Screenshots / GIF

![Screenshot 2023-03-29 at 4 17 53 PM](https://user-images.githubusercontent.com/97971327/228657472-13a083b1-fea7-4a09-ab7e-f7e55ef968b4.png)

## How To Test

1. Compare path values between "star.svg" and "star-empty.svg" to ensure they are the same.
2. Checkout out branch code, do a build, and ensure they are aligned.
